health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/AssumptionsForm.js

```
@@ -168,38 +168,38 @@ const AssumptionsForm = ({ onSelect, ...props }) => {
+                        size={"large"}
+                        variation={"secondary"}
+                        label={t("MP_ACK")}
+                        onClick={() => {
+                            setShowPopup(false);
+                            //   setCanUpdate(true);
+                        }}
+                    />
+                ]}
             // sortFooterChildren={true}
-          ></PopUp>}
+            ></PopUp>}
```

Biome

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)